book/src/drivers.md

```diff
@@ -62,7 +62,8 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [HPE ClusterStor Lustre CSI](https://github.com/HewlettPackard/lustre-csi-driver) | `lustre-csi.hpe.com` | v1.5 | A Container Storage Interface (CSI) Driver for HPE Cray ClusterStor Lustre Storage | Persistent | Read/Write Multiple Pods | No |
 [HPE Ezmeral (MapR)](https://github.com/mapr/mapr-csi) | `com.mapr.csi-kdf` | v1.3 | A Container Storage Interface (CSI) Driver for HPE Ezmeral Data Fabric | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion, Cloning
 [Huawei Storage CSI](https://github.com/Huawei/eSDK_K8S_Plugin) | `csi.huawei.com` | v1.0, v1.1, v1.2 | A Container Storage Interface (CSI) Driver for FusionStorage, OceanStor 100D, OceanStor Pacific, OceanStor Dorado V3, OceanStor Dorado V6, OceanStor V3, OceanStor V5 | Persistent | Read/Write Multiple Pod | Yes | Snapshot, Expansion, Cloning
-[HyperV CSI](https://github.com/Zetanova/hyperv-csi-driver) | `eu.zetanova.csi.hyperv` | v1.0, v1.1 | A Container Storage Interface (CSI) driver to manage hyperv hosts | Persistent | Read/Write Multiple Pods | Yes | 
+[HwameiStor](https://github.com/hwameistor/hwameistor) | `lvm.hwameistor.io disk.hwameistor.io ` | v1.3 | A Container Storage Interface (CSI) Driver for Local Storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Expansion, LVM HA 
```

Comment:
"LVM HA" is not a CSI spec feature. Please remove it from this column.

Reply:
@xing-yang removed.